example/src/useAxios/DefaultParameter.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Container from '../Container';
+import Heading from '../Heading';
+import TextBlock from '../TextBlock';
+import { useAxios } from '../../../src';
+
+interface Data {
+  data: {
+    name: string;
+    color: string;
+  };
+}
+
+export default () => {
+  const { data = { data: { name: 'salmon', color: '#FA8072' } }, error, loading } = useAxios<Data>(
+    'https://reqres.in/api/bad/request'
+  );
+
+  return (
+    <Container>
+      <Heading>useAxios</Heading>
+
+      <TextBlock>
+        {loading && 'Loading...'}
+        {error && !data && error.message}
```

Comment:
Won't `data` always coerce to `true` here?

Reply:
Without this extra check, both the `error` and `data` render:

![image](https://user-images.githubusercontent.com/28536523/68168146-726f0380-ff2d-11e9-98f5-336c87f11ffd.png)

Reply:
You should be able to remove the `!data` part though, since it will always be truthy anyway.